File: College-Sports-WebApp/Database/Models/Event.cs
using System.Collections.Generic; 
namespace College_Sports_WebApp.Database.Models{ 

    public class Event
    {
        public string id { get; set; }
        public string uid { get; set; }
        public string date { get; set; }
        public string name { get; set; }
        public string shortName { get; set; }
        public Season season { get; set; }
        public List<Competition> competitions { get; set; }
        public List<Link> links { get; set; }
        public Status status { get; set; }
        public Weather weather { get; set; }
    }

}

File: src/app/models/event.ts
/* tslint:disable */
/* eslint-disable */
import { Competition } from './competition';
import { Link } from './link';
import { Season } from './season';
import { Status } from './status';
import { Weather } from './weather';
export interface Event {
  competitions?: Array<Competition> | null;
  date?: string | null;
  id?: number;
  links?: Array<Link> | null;
  name?: string | null;
  season?: Season;
  shortName?: string | null;
  status?: Status;
  uid?: string | null;
  weather?: Weather;
}
